metdata.yaml

```
@@ -0,0 +1,28 @@
+  group: ${{ github.repository }}-${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  validate-metadata:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Validate asd
+        uses: asdasd
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)